stubs/psycopg2/psycopg2/_psycopg.pyi

```diff
@@ -371,64 +371,99 @@ class connection:
     ProgrammingError: Any
     Warning: Any
     @property
-    def async_(self) -> Any: ...
-    autocommit: Any
+    def async_(self) -> int: ...
+    autocommit: bool
     @property
     def binary_types(self) -> Any: ...
     @property
-    def closed(self) -> Any: ...
+    def closed(self) -> int: ...
     cursor_factory: Callable[..., _cursor]
-    deferrable: Any
     @property
-    def dsn(self) -> Any: ...
+    def dsn(self) -> str: ...
     @property
-    def encoding(self) -> Any: ...
+    def encoding(self) -> str: ...
     @property
     def info(self) -> ConnectionInfo: ...
-    isolation_level: Any
-    notices: Any
-    notifies: Any
     @property
-    def pgconn_ptr(self) -> Any: ...
+    def isolation_level(self) -> int | None: ...
+    @isolation_level.setter
+    def isolation_level(self, __value: str | bytes | int | None) -> None: ...  # pyright: ignore
```

Comment:
I'm fine with using `# pyright: ignore` here (rather than a bare `# type: ignore`, which would be the other possibility), but I think some other maintainers had some concerns when we last discussed it in #7497. @srittau suggested in https://github.com/python/typeshed/issues/7497#issuecomment-1070817560 that we should maybe just disable this specific pyright check in our `pyrightconfig.json`, since it's the only type checker in our CI that complains about it, and it basically just surfaces issues in the implementation rather than the stubs.

Reply:
Good point, #7985